### PR TITLE
Add template for ELK on CentOS

### PR DESCRIPTION
**Pull request by lizzha**
Add template for ELK on CentOS

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@lizzha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (lizzha). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@lizzha Why create a new template instead of adding the new os choice to the existing template? https://github.com/Azure/azure-quickstart-templates/tree/master/diagnostics-with-elk

**Comment by lizzha** [comment]
@singhkay this one is more generic than diagnostics-with-elk, it allows user to configure whether to install plugin and which plugin to install
while diagnostics-with-elk installs the logstash-input-azurewadtable plugin

File: elk-centos/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+        "storageAccountType": "Standard_LRS",
+        "publicIPAddressType": "Dynamic",
+        "vmStorageAccountContainerName": "vhds",
+        "vmName": "logstashvm1",
+        "vmSize": "Standard_D1",
+        "virtualNetworkName": "es-vnet",
+        "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+        "subnetRef": "[concat(resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworkName')), '/subnets/', 'other')]",
+        "esHost":  "10.0.2.100",
+        "esTemplate":  "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/elasticsearch/azuredeploy.json"
```
Comment (bmoore-msft) [inline elk-centos/azuredeploy.json]: should be using _artifactsLocation

File: elk-centos/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+            }
+        },
+        "vmDataNodeCount": {
+            "type": "int",
+            "defaultValue": 1,
+            "metadata": {
+                "description": "Number of Elasticsearch data nodes"
+            }
+        },
+        "vmSizeMasterNodes": {
```
Comment (bmoore-msft) [inline elk-centos/azuredeploy.json]: move size params to vars

**Comment by bmoore-msft** [comment]
If this template is a superset - can we retire the other sample?

**Pull request closed without merging**